cmd/oceantv/broadcast_machine.go

```diff
@@ -164,6 +166,24 @@ func (sm *broadcastStateMachine) handleGoodHealthEvent(event goodHealthEvent) er
 	return nil
 }
 
+func (sm *broadcastStateMachine) handleControllerFailureEvent(event controllerFailureEvent) error {
+	sm.log("handling controller failure event")
+	switch sm.currentState.(type) {
+	case *directStarting:
+		onFailureClosure(sm.ctx, sm.ctx.cfg, true)(fmt.Errorf("Controller not reporting. Likely caused by low voltage."))
```

Comment:
Each onFailureClosure call uses the same error; consider defining it once for use by all of these.

Reply:
Was there somewhere these types of errors are being defined in a file, couldn't find any others. Otherwise I'll just put it at the tip of this file

Reply:
Just put at top of this method I reckon. Also, you don't need to use fmt.Errorf considering you're not providing any additional arguments for formatting.

Reply:
Done.